Components/Alert/alert.css

```diff
@@ -0,0 +1,66 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --theme-green: rgba(34, 197, 94, 0.4);
+  --theme-gray: rgba(15, 28, 39, 0.1);
+  --theme-yellow: rgba(253, 224, 71, 0.4);
+  --theme-red: rgba(239, 68, 68, 0.4);
+  --theme-blue: rgba(56, 189, 248, 0.4);
+  --theme-purple: rgba(99, 102, 241, 0.2);
+  --theme-black: rgb(16, 14, 9);
+}

```

Comment:
I saw you are copying your CSS variables and reset code in every file. Instead of doing this, you can create a separate file where you can put this code. Then import that file wherever you need that code.

Reply:
Moved common CSS to default.css, refactored the files accordingly